runtime/compiler/env/ProcessorDetection.cpp

```diff
@@ -648,10 +648,12 @@ void
 TR_J9VM::initializeProcessorType()
    {
    TR_ASSERT(_compInfo,"compInfo not defined");
-   TR::Compiler->target.cpu.applyUserOptions();
 
    if (TR::Compiler->target.cpu.isZ())
       {
+      TR::Compiler->target.cpu.applyUserOptions();
```

Comment:
I don't see `applyUserOptions` for x86

Reply:
I've put `applyUserOptions()` back to the original place. There is no need to have this change here.